test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/RazorEngineTest.cs

```diff
@@ -131,13 +130,14 @@ private static void RunRuntimeTest(string testName)
             });
 
             var inputContent = ResourceFile.ReadResource(_assembly, inputFile, sourceFile: true);
-            var item = new TestRazorProjectItem(inputFile) { Content = inputContent, };
+            var item = new TestRazorProjectItem("/" + inputFile) { Content = inputContent, };
             var project = new TestRazorProject(new List<RazorProjectItem>()
             {
                 item,
             });
 
             var razorTemplateEngine = new MvcRazorTemplateEngine(engine, project);
+            razorTemplateEngine.Options.ImportsFileName = "_ViewImports.cshtml";
```

Comment:
Although this doesn't matter, I am leaving these here because it is better to test with this set.